cardano-cli/src/Cardano/CLI/Json/Friendly.hs

```diff
@@ -420,15 +433,15 @@ renderCertificate sbe = \case
             [ "stake pool key hash" .= String (textShow kh)
             , "epoch" .= epoch
             ]
-        Ledger.DelegStakeTxCert stakeCredential (Shelley.KeyHash kh) ->
-          "Stake address delegation" .= object
-            [ "stake credential" .= stakeCredJson sbe stakeCredential
-            , "key hash" .= String (textShow kh)
+        ConwayLedger.UpdateDRepTxCert drepCredential mbAnchor ->
+          "Drep certificate update" .= object
+            [ "Drep credential" .= drepCredJson sbe drepCredential
+            , "anchor " .= mbAnchor
```

Comment:
This case expression and a ledger `COMPLETE` pragma currently disagree, only on this case.

I currently include a commit in this PR that assumes the `COMPLETE` pragma is right and the case expression was wrong.

But I'm not sure what exactly the body of this `UpdateDRepTxCert` should be. For example, should it use `conwayToObject` instead of the `drepCredJson` I added?

Reply:
On Slack, Alexey wrote "yes, complete pragma is right. `UpdateDRepTxCert` is the new addition".